Northstar.Custom/mod/scripts/vscripts/sh_message_utils.gnut

```diff
@@ -168,6 +202,46 @@ int function NSGetPlayerResponse( entity player )
 	return server.playerPollResponses[ player ] - 1
 }
 
+string function NSCreateChoiceOnAllPlayers( string option1, string option2, float duration )
```

Comment:
Inputs should be an array so that also a single option can be passed
(or more than two once we got RUI stuff better figured out :eyes:)

Reply:
Adressed in 7a8b711ce50b1f6277276ffdb573d09a310333d8